rfcs/tag-component.md

```diff
@@ -0,0 +1,96 @@
+# Tag Component
+
+### Exports
+
+- `StatefulTag`
+- `StatefulTagContainer`
+- `Tag`
+- `StyledRoot`
+- `StyledCross`
+
+### `Tag` API
+
+- `style: STYLE.blue | STYLE.orange | STYLE.green | STYLE.red`:
```

Comment:
`style` is a standard HTML attribute. Can we call it something else? 

Also, can we take the colors from the [theme](https://github.com/uber-web/baseui/blob/master/src/themes/light-theme.js)?

Reply:
ok, called `color`